Swiftfin tvOS/Components/SplitFormWindowView.swift

```diff
@@ -31,6 +31,7 @@ struct SplitFormWindowView: View {
             .if(descriptionTopPadding) { view in
                 view.padding(.top)
             }
+            .scrollClipDisabled()
```

Comment:
This does fix the issue like I said earlier but now introduces a different clipping behavior for long lists when cells clip out. The documentation comment for this modifier has a solution for custom clipping behavior so we'll have to take a look at something like that later with a desirable gradient mask.

Reply:
Sorry for my confusion. Do you want me to leave this in, take this out, or would you look me to look into custom clipping behavior? Happy to help out wherever I can!

Reply:
Leave this in to fix the original clipping issue. The new clipping issue was something I just noticed while testing this and we can fix later.